gRPC-C++.podspec

```diff
@@ -23,15 +23,15 @@
 Pod::Spec.new do |s|
   s.name     = 'gRPC-C++'
   # TODO (mxyan): use version that match gRPC version when pod is stabilized
-  # version = '1.19.0'
+  # version = '1.19.1'
```

Comment:
Nit: looks like we are already using grpc_version so this line and the previous comment line should be removed (and the generation script modified not to look for this)

Reply:
cc @muxi 
This can be resolved in master for next release.